.gitlab-ci.yml

```diff
@@ -72,34 +74,40 @@ test:static:
 
 test:lint:
   stage: test
+  image: node
   script:
-    - docker build -t gui-test-image -f Dockerfile.testing .
-    - docker run --rm gui-test-image
+    - npm ci
+    - npm run lint
   except:
     - master
-  tags:
-    - docker
 
 test:acceptance:
   image: tiangolo/docker-with-compose
+  dependencies:
+    - build
   services:
     - docker:18-dind
-  stage: test
+  stage: e2e-test
   script:
+    - docker load -i image.tar
+    - docker tag $SERVICE_IMAGE $DOCKER_REPOSITORY:pr
     - apk add git bash
-    - git clone https://github.com/mendersoftware/integration.git
+    - git clone https://github.com/mzedel/integration.git --branch fix/e2e-test-ci-setup --single-branch integration
```

Comment:
Remember remove the `--branch` parameter(s) before merging to master

Reply:
Thanks, it should be in line with the proper repo again👌